.github/workflows/publish.yml

```diff
@@ -1,10 +1,10 @@
----
+## Reference: https://github.com/helm/chart-releaser-action
 name: Chart Publish
 on:
   push:
     branches:
       - main
-      - rewrite-build
```

Comment:
leftover of @oliverbaehler during the implementation of PR #741

https://github.com/argoproj/argo-helm/pull/741/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R7